src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -628,6 +640,33 @@ public String getDisplayName() {
         return super.getDisplayName();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ACL getACL() {
+        final ACL acl = super.getACL();
+        if (getParent() instanceof ComputedFolder<?>) {
```

Comment:
Changed Behavior:

If the OrganizationFolder is a direct child of a ComputedFolder, do not allow direct configuration or deletion of the Folder, nor creation, modification, or deletion of Views.

Reply:
Sam logic should also be applied to MultiBranchProject itself, i.e. if it is a direct child of a ComputedFolder, rather than hardcoded to OrganizationFolder https://github.com/jenkinsci/branch-api-plugin/blob/ed229d8e1392645c99e5686b8bd652a14ec87f4a/src/main/java/jenkins/branch/MultiBranchProject.java#L772